internal/pkg/service/buffer/worker/service/service.go

```diff
@@ -0,0 +1,17 @@
+package service
+
+import (
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/worker/dependencies"
+)
+
+type Service struct {
```

Comment:
Vytvoril som worker service, miesto, kde bude implementacia.

Po spusteni to vyzera nejak takto:
![image](https://user-images.githubusercontent.com/19371734/203275821-9f72da4b-0975-4127-a4fd-7d565a34fcd2.png)